<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilmWidget</class>
 <widget class="QWidget" name="FilmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="imagLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>491</width>
     <height>701</height>
    </rect>
   </property>
   <property name="text">
    <string>imagLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="fnameLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>电影名称:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="introductionBrowser">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>370</y>
     <width>371</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="introductionLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>介绍:</string>
   </property>
  </widget>
  <widget class="QLabel" name="durationLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>190</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>时长:</string>
   </property>
  </widget>
  <widget class="QLabel" name="fnameLabel_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>60</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>电影名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="durationLabel_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>时长</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
